Server.MirForms/Database/GameShop.designer.cs

```diff
@@ -444,5 +470,7 @@ private void InitializeComponent()
         private System.Windows.Forms.ComboBox CategoryFilter_lb;
         private System.Windows.Forms.Button ResetFilter_button;
         private System.Windows.Forms.Button ServerLog_button;
+        private System.Windows.Forms.CheckBox checkBox2;
+        private System.Windows.Forms.CheckBox checkBox1;
```

Comment:
please name checkBox1 and checkBox2 on the gameshop form so they're abit more friendly